### PR TITLE
Replace Companies House Parent version to 2.1.6

### PR DESCRIPTION
**Pull request by aduffy-ch**
https://companieshouse.atlassian.net/browse/JU-530

Replace Companies House Parent version to 2.1.6
Upgrade Spring Boot Version to at least 3.3.2
Make sure the Jib plugin docker images are correct
Implement new Suppressions method

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: pom.xml [hunk 1]
```diff
@@ -132,7 +134,7 @@
 <!--			Overriding spring-web version to remove vulnerability-->
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-web</artifactId>
-			<version>6.1.6</version>
```
Comment (vshastri-ch) [inline pom.xml:135]: You can remove this spring-web dependency safely :)

**Review by vshastri-ch — DISMISSED** [review]

**Review by foman-ch — DISMISSED** [review]

**Review by vshastri-ch — DISMISSED** [review]

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by akunati-ch — APPROVED** [review]

**Pull request merged**